outside.go

```diff
@@ -7,21 +7,21 @@ import (
 	"time"
 
 	"github.com/flynn/noise"
+	"github.com/golang/protobuf/proto"
 	"github.com/sirupsen/logrus"
 	"github.com/slackhq/nebula/cert"
 	"github.com/slackhq/nebula/firewall"
 	"github.com/slackhq/nebula/header"
 	"github.com/slackhq/nebula/iputil"
 	"github.com/slackhq/nebula/udp"
 	"golang.org/x/net/ipv4"
-	"google.golang.org/protobuf/proto"
 )
```

Comment:
I had a build error with the new protobuf import. I didn't realize the `google.golang.org` one was deprecated - this might be something wrong with my local protobuf compiler, will look into it.
```
./outside.go:89:31: cannot use &m (value of type *NebulaControl) as type protoreflect.ProtoMessage in argument to proto.Marshal:
        *NebulaControl does not implement protoreflect.ProtoMessage (missing ProtoReflect method)
```

Reply:
I read up on this a little, and this is what I think is going on - 

The cert proto uses the new-and-improved protobuf generation code
https://github.com/slackhq/nebula/blob/master/cert/Makefile#L5
But the messages themselves use old-style protobufs
https://github.com/slackhq/nebula/blob/master/Makefile#L151

I tried to update the Nebula messages to the new style, but ran into a decision point due to the lack of MarshalTo() in the new style protobufs (which is called a couple of times in lighthouse.go) - at which point I stopped trying the migration, as it looks large enough to be a separate task.

I conclude that I can either move the Nebula protocol protobuf interaction out of this file (so that the existing Cert protobuf code can use the new imported module as-was), or import both protobufs and give one an alias, since this file handles both Nebula protocol protobufs as well as cert protobufs.